source/user-portal/scripts/upload.js

```diff
@@ -0,0 +1,47 @@
+
+
+function init() {
+
+}
+
+
+var storageFiles = JSON.parse(localStorage.getItem("storageFiles")) || {},
+    user = document.getElementById("user");
+
+if(storageFiles.user != null){
+    user.src = storageFiles.user;
+}
+
+window.addEventListener('load', function() {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8460275)
```suggestion
window.addEventListener("load", function() {
```